bin/mark_outliers_in_json

```diff
@@ -21,11 +21,13 @@
 """
 
 import argparse
-import bz2
-import json
-import numpy
 import os
 import os.path
+import sys
+
+sys.path.insert(0, '../warmup')
```

Comment:
I am a bit sceptical about this. I can never remember the details either, but I think it adds the path relative to the current working directory, so the script won't work if ran from somewhere else. you could turn this into an absolute path relative to the file, with code roughly like this:

``os.path.join(os.path.dirname(os.path.abspath(__file__)), "..", "warmup")``